src/db/appsheet_api.py

```diff
@@ -111,10 +119,12 @@ def add_aircraft(self, regno, test=False, description=""):
         try:
             if not self.use_fake_calls:
                 ret = self.sendop(self.config.private_vars["appsheet"]["aircraft_url"], body)
+                appsheet_error_gauge.labels('appsheet add_aircraft error', "").set(0)
                 return ret["Rows"][0]["Row ID"]
             else:
                 return FAKE_KEY
         except Exception as e:
+            appsheet_error_gauge.labels("appsheet add_aircraft error", str(e)).set(1)
```

Comment:
Should it be .inc() instead?

Reply:
Yeah, was supposed to be a trigger but setting and resetting it is awkward and might as well use the syslogs